.ci/Jenkinsfile

```diff
@@ -78,8 +78,10 @@ pipeline {
             withGithubNotify(context: 'Sanity checks', tab: 'tests') {
               deleteDir()
               unstash 'source'
-              dir("${BASE_DIR}"){
-                preCommit(commit: "${GIT_BASE_COMMIT}", junit: true)
+              dir(BASE_DIR){
+                sshagent(['f6c7695a-671e-4f4f-a331-acdce44ff9ba']) {
```

Comment:
why running the preCommit in an SSH agent?

Reply:
sshagent is required as long as https://github.com/elastic/apm-integration-testing/pull/571/files#diff-82c3b36709926de3a1b27bea0d8258ccR39 is a private repo.

Not sure yet whether those implementation details should be defined within the `preCommit` step to keep the pipeline simple or just wait until the private-repo has been set.  That's the reason I have not set ready this PR yet, but wanted to PoC this approach worked in other repos.

Reply:
Already removed from the pipeline itself, as it's now provided within the `preCommit` step then once we open the library we can deprecate that particular behavior without changing any pipelines.